src/Mpociot/ApiDoc/Generators/AbstractGenerator.php

```diff
@@ -135,17 +135,16 @@ protected function getRouteRules($route)
 
         foreach ($reflectionMethod->getParameters() as $parameter) {
             $parameterType = $parameter->getClass();
-            if (! is_null($parameterType) && class_exists($parameterType->name)) {
-                $className = $parameterType->name;
-                $parameterReflection = new $className;
-                if ($parameterReflection instanceof FormRequest) {
-                    if (method_exists($parameterReflection, 'validator')) {
-                        return $parameterReflection->validator()->getRules();
-                    } else {
-                        return $parameterReflection->rules();
-                    }
-                }
+            if (is_null($parameterType) && ! class_exists($parameterType->name)) {
+                continue;
             }
+            $className = $parameterType->name;
+            $parameterReflection = new $className;
+            if ($parameterReflection instanceof FormRequest && method_exists($parameterReflection, 'validator')) {
+                return $parameterReflection->validator()->getRules();
+            }
+
+            return $parameterReflection->rules();
```

Comment:
This will break, because it will try to call the `rules` method on every object, even if it's not an instance of `FormRequest`